packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/06-deeplinking.mdx

```diff
@@ -1,194 +1,193 @@
 ---
-id: deeplinking
 title: Deep Linking
+description: Tutorial to integrate deep linking for the calls
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-import XCodeInfoUrlTypesImage from '../assets/06-advanced/deeplinking/xcode-info-url-types.png';
-import XCodeSigningCapabilities from '../assets/06-advanced/deeplinking/xcode-signing-capabilitites.png';
+This guide talks about deep linking into a [video call](../02-tutorials/01-video-calling.mdx) or a generic application directly. It also focuses on getting the call id or similar entity through a Deep link URL and thereby use it to start a call.
 
-## Implementing Call Deep Linking
+Our API makes it very easy to integrate and start with your custom UI, or our existing screens powered by our UI components.
```

Comment:
I am not sure why this sentence is necessary in a deeplinking guide where we use no API of stream

Reply:
Yes, I will remove it